lib/rucio/rse/rsemanager.py

```diff
@@ -713,6 +716,31 @@ def find_matching_scheme(rse_settings_dest, rse_settings_src, operation_src, ope
     raise exception.RSEProtocolNotSupported('No protocol for provided settings found : %s.' % str(rse_settings_dest))
 
 
+def _retry_protocol_stat(protocol, pfn):
+    """
+    try to stat file, on fail try again 1s, 2s, 4s, 8s, 16s, 32s later. Fail is all fail
+
+    :param protocol     The protocol to use to reach this file
+    :param pfn          Physical file name of the target for the protocol stat
+    """
+    from time import sleep
+    try:
+        retries = config_get('client', 'protocol_stat_retries')
```

Comment:
The `config_get` command already has a default option. Would shorten the code a bit to do that. Also the import should go to the top of the file. (I think this was already fixed in your previous PR, maybe in the re-creation as patch this got lost?)